### DISCUSSION TITLE
Final in the middle

### DISCUSSION THREAD

**Discussion by tarung02** [Drarig29/brackets-viewer.js#107] [Q&A] [answered]
Hi,

Is it possible to have the final in the middle and the semi-finals leading in from each side?

thanks.

**Comment by Drarig29** [comment, marked as answer]
This sounds like https://github.com/Drarig29/brackets-viewer.js/discussions/106, and it's not supported.

**Reply by tarung02** [reply]
Thank you and you're right. It is the same question.
